code/controllers/subsystem/research.dm

```diff
@@ -294,7 +297,8 @@ SUBSYSTEM_DEF(research)
 	var/list/errored_datums = list()
 	var/list/point_types = list()				//typecache style type = TRUE list
 	//----------------------------------------------
-	var/list/single_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 3.5)	//citadel edit - techwebs nerf
+	var/list/BOSsingle_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 3.5)	//citadel edit - techwebs nerf
+	var/list/VAULTsingle_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 35)
```

Comment:
The vault has 10x the research of the BoS? Yeah, right...

Reply:
BoS has other ways to get research now.